README.md

```diff
@@ -1,95 +1,100 @@
+```yaml
 packages:
   - package: fivetran/jira_source
-    version: [">=0.4.0", "<0.5.0"]
+    version: [">=0.5.0", "<0.6.0"]
 ```
-
-Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
-
-## Configuration
-By default, this package looks for your Jira data in the `jira` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your Jira data is, add the following configuration to your `dbt_project.yml` file:
+## Step 3: Configure Your Variables
+### Database and Schema Variables
+By default, this package will run using your target database and the `jira` schema. If this is not where your Jira data is (perhaps your Jira schema is `jira_fivetran`), add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
-
-...
-config-version: 2
-
 vars:
     jira_database: your_database_name
     jira_schema: your_schema_name 
 ```
-
-### Disabling models
-
-It's possible that your Jira connector does not sync every table that this package expects. If your syncs exclude certain tables, it is because you either don't use that functionality in Jira or actively excluded some tables from your syncs. To disable the corresponding functionality in the package, you must add the relevant variables. By default, all variables are assumed to be `true`. Add variables for only the tables you would like to disable:  
+### Disabling Components
+Your Jira connector might not sync every table that this package expects. If you do not have the `SPRINT`, `COMPONENT`, or `VERSION` tables synced, add the following variable to your root `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
-
-...
-config-version: 2
-
 vars:
     jira_using_sprints: false   # Disable if you do not have the sprint table, or if you do not want sprint related metrics reported
     jira_using_components: false # Disable if you do not have the component table, or if you do not want component related metrics reported
     jira_using_versions: false # Disable if you do not have the versions table, or if you do not want versions related metrics reported
 ```
-
-### Changing the Build Schema
-By default this package will build the Jira staging models within a schema titled (<target_schema> + `_stg_jira`) in your target database. If this is not where you would like your Jira staging data to be written to, add the following configuration to your `dbt_project.yml` file:
+## (Optional) Step 4: Additional Configurations
+### Change the Build Schema
+By default, this package builds the GitHub staging models within a schema titled (<target_schema> + `_stg_jira`) in your target database. If this is not where you would like your GitHub staging data to be written to, add the following configuration to your root `dbt_project.yml` file:
```

Comment:
Jira, not Github?

Reply:
Yeah this should be Jira and not github.